pallets/ddc-verification/src/lib.rs

```diff
@@ -753,30 +753,23 @@ pub mod pallet {
 	}
 
 	#[derive(Debug, Clone, PartialEq)]
-	pub(crate) struct ConsistentGroup<A: Aggregate>(pub ActivityHash, pub Vec<A>);
-	impl<A: Aggregate> ConsistentGroup<A> {
-		pub fn hash(&self) -> ActivityHash {
-			self.0
-		}
-
-		pub fn get(&self, idx: usize) -> Option<&A> {
-			self.1.get(idx)
-		}
-
-		pub fn len(&self) -> usize {
-			self.1.len()
-		}
+	pub(crate) struct ConsolidatedAggregate<A: Aggregate> {
```

Comment:
Would be nice to add a comment with an explanation on what "consolidated" means.

Reply:
In-code docs added